src/main/java/io/fusionauth/load/JavaHTTPLoadTestWorker.java

```diff
@@ -26,25 +34,62 @@
  * @author Daniel DeGroff
  */
 public class JavaHTTPLoadTestWorker extends BaseWorker {
+  private final HttpClient javaHTTPClient = HttpClient.newBuilder()
```

Comment:
Better to do this initialization in a constructor? I'm not a fan of having anything that feels like logic outside of a method, but that's kind of just personal preference.